### ISSUE TITLE
Convert gprDemoArd to python

### ISSUE THREAD

**Issue by mjsML** [probml/pyprobml#217]
This is figure number 17.2 in the book. its matlab code is [here](https://github.com/probml/pmtk3/blob/master/demos/gprDemoArd.m)

**Comment by ashishpapanai** [comment]
Working on it.

**Comment by ashishpapanai** [comment]
@mjsML, I am getting this not so similar image by converting the code. The plotted values look the same in the MATLAB and python version. 

MATLAB Version: 
```
L1 = 1; L2 = 5;
%Q = Q + (repmat(x(:,1),1,n)-repmat(x(:,1)',n,1)).^(2);
Q = Q + (repmat(x(:,1),1,n)-repmat(x(:,1)',n,1)).^(2) / (L1^2);
%Q = Q + (repmat(x(:,2),1,n)-repmat(x(:,2)',n,1)).^2/9;
Q = Q + (repmat(x(:,2),1,n)-repmat(x(:,2)',n,1)).^2 /(L2^2);
Q = exp(-0.5*Q);
y = chol(Q+1e-9*eye(n))'*randn(n,1);
```
chol in MATLAB is giving this error `error: chol: input matrix must be positive definite`

The Python Implementation. 
```
Q = np.zeros((n, n))

for d in range(0, D):
    Q = Q + np.power(np.tile((x[:, d]).reshape(n, 1), (1, n)) -
                     (np.tile((x[:, d]).reshape(1, n), (n, 1))), 2)


Q = np.exp(np.multiply(-0.5, Q))

arr = np.transpose(Q+(1e-9)*np.eye(n))
y = np.dot(cholesky((arr)), np.random.randn(n, 1))

```

![GPARD2](https://user-images.githubusercontent.com/52123364/112540022-b4753f80-8dd7-11eb-8d01-c220b74487de.png)

**Comment by mjsML** [comment]
closed by #293
